include/mata/afa.hh

```diff
@@ -63,7 +64,7 @@ using StringDict = Mata::Nfa::StringMap;
 using StateSet = OrdVec<State>;
 using StateClosedSet = Mata::ClosedSet<Mata::Afa::State>;
 
-using Alphabet = Mata::Nfa::Alphabet;
+using Alphabet = Mata::Alphabet;
```

Comment:
We should probably just rename all occurrences of `Alphabet` to `Mata::Alphabet` and do not introduce new type definitions in header files.

Reply:
Solved.